src/main.rs

```diff
@@ -23,6 +25,7 @@ struct Cli {
 
 #[derive(Debug, Subcommand)]
 enum Commands {
+    Init(ShellName),
```

Comment:
Add docstring

Reply:
I don't really like 'Init' for this. A more descriptive command name would be better imo, eg 'Completions'